pub-critic/src/app/components/Filter/Filter.tsx

```diff
@@ -87,6 +107,9 @@ export const Filter = ({
             onSelect={(value) => {
               setGenre(value as number);
             }}
+            initSelected={
+              genres.find((selectedGenre) => selectedGenre.id == genre)?.name
```

Comment:
Maybe is better for performance to move these calculations outside of rendering part. Also will be easier to read.

Reply:
:white_check_mark: